src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting, including text, contained inside the element, requires 2-space offset. Also, blank line shouldn't be between parent and child elements.